tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -767,6 +767,85 @@ var _ = Describe("Create machinepool",
 
 				})
 
+			It("will validate machine pool creation - [id:38775]",
+				labels.Runtime.Day2, labels.Medium,
+				func() {
+					mpName := "mp-38775"
+					fakeClusterName := "fake_38775"
+					By("Create machine pool with no flags")
+					output, err := rosaClient.MachinePool.CreateMachinePoolEmpty()
+					Expect(err).To(HaveOccurred())
+					Expect(output.String()).Should(ContainSubstring("required flag(s) \"cluster\" not set"))
+
+					By("Create machine pool with invalid values")
```

Comment:
Better to indicate the validation checkpoint in `By`
```suggestion
					By("Create machine pool to non-exist cluster")
```